Kernel/Syscalls/thread.cpp

```diff
@@ -197,7 +197,7 @@ ErrorOr<FlatPtr> Process::sys$set_thread_name(pid_t tid, Userspace<char const*>
 
 ErrorOr<FlatPtr> Process::sys$get_thread_name(pid_t tid, Userspace<char*> buffer, size_t buffer_size)
 {
-    VERIFY_PROCESS_BIG_LOCK_ACQUIRED(this);
+    VERIFY_NO_PROCESS_BIG_LOCK(this);
```

Comment:
The flow of this syscall doesn't look right. Ofc, I am not expecting you to fix this now, I just say it so we all know that it's wrong to do:
```c++
SpinlockLocker locker(thread->get_lock());
auto thread_name = thread->name();
```
Instead of just having `Thread::name()` to return a `SpinlockProtected&` value.

Reply:
I might as well make this change while I'm here.